### ISSUE TITLE
Sonarqube failing to start on QNap NAS.

### ISSUE THREAD

**Issue by KelSolaar** [SonarSource/docker-sonarqube#187]
Hi,

I'm trying to run Sonarqube on my QNap NAS, the container is created successfully but upon running I get a ton of `chown` errors:

```
[...]
chown: changing ownership of '/opt/sonarqube/web/js': Operation not permitted
chown: changing ownership of '/opt/sonarqube/web/mstile-512x512.png': Operation not permitted
chown: changing ownership of '/opt/sonarqube/web/robots.txt': Operation not permitted
chown: changing ownership of '/opt/sonarqube/web': Operation not permitted
chown: changing ownership of '/opt/sonarqube': Operation not permitted
```

Those are happening when the following line is executed: https://github.com/SonarSource/docker-sonarqube/blob/5d695dae7e2c5565e6dd09991a719fae5f5cefec/7.1/run.sh#L9

I'm a bit unclear why it does that though. Obviously, spawning a bash shell in the container (`docker run -it --rm sonarqube bash`) and running the chown command (`chown -R sonarqube:sonarqube $SONARQUBE_HOME`) yields the same result.

The container works perfectly fine on my laptop. Any ideas are appreciated!

Cheers,

Thomas

**Comment by allustin** [comment]
@KelSolaar it docker issue. SELinux  i think with mount volume. Does you mount something ?

**Comment by janos-ss** [comment]
We have just released a new build of 7.4. The `run.sh` in this build doesn't have this line anymore. Please give that a try and let us know.
